demo/openssl-qat-engine/Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN cd /QAT_Engine && \
     --with-multibuff_install_dir=/usr/local \
     --with-qat_engine_id=qat-sw && \
     make && make install && \
+    mv /usr/lib64/engines-1.1/qat-sw.so /usr/lib64/engines-1.1/qat-sw.so.tmp && \
+    make clean && \
+    mv /usr/lib64/engines-1.1/qat-sw.so.tmp /usr/lib64/engines-1.1/qat-sw.so && \
```

Comment:
Looks good. I guess a comment would be helpful here why this dancing is needed.

Reply:
a comment added